yarn-project/acir-simulator/src/acvm/acvm.ts

```diff
@@ -69,26 +69,32 @@ export interface ACIRExecutionResult {
  * The function call that executes an ACIR.
  */
 export async function acvm(
+  backend: SimulatedBackend,
   acir: Buffer,
   initialWitness: ACVMWitness,
   callback: ACIRCallback,
 ): Promise<ACIRExecutionResult> {
   const logger = createDebugLogger('aztec:simulator:acvm');
-  const partialWitness = await executeCircuit(acir, initialWitness, async (name: string, args: ForeignCallInput[]) => {
-    try {
-      logger(`Oracle callback ${name}`);
-      const oracleFunction = callback[name as ORACLE_NAMES];
-      if (!oracleFunction) {
-        throw new Error(`Oracle callback ${name} not found`);
+  const partialWitness = await executeCircuit(
+    backend,
+    acir,
+    initialWitness,
+    async (name: string, args: ForeignCallInput[]) => {
+      try {
+        logger(`Oracle callback ${name}`);
+        const oracleFunction = callback[name as ORACLE_NAMES];
+        if (!oracleFunction) {
+          throw new Error(`Oracle callback ${name} not found`);
+        }
+
+        const result = await oracleFunction.call(callback, ...args);
+        return [result];
+      } catch (err: any) {
+        logger(`Error in oracle callback ${name}: ${err.message ?? err ?? 'Unknown'}`);
+        throw err;
       }
-
-      const result = await oracleFunction.call(callback, ...args);
-      return [result];
-    } catch (err: any) {
-      logger(`Error in oracle callback ${name}: ${err.message ?? err ?? 'Unknown'}`);
-      throw err;
-    }
-  });
+    },
+  );
```

Comment:
This is mainly autoformatting. All I did was add the backend arg to `acvm()` and `executeCircuit()`.